Formula/bats-core.rb

```diff
@@ -0,0 +1,53 @@
+
+class BatsCore < Formula
+  desc "Bash Automated Testing System"
+  homepage "https://github.com/bats-core/bats-core"
+  url "https://github.com/bats-core/bats-core/archive/v0.4.0.tar.gz"
+  sha256 "e3b65b50a26e3f0c33b5d0a57d74101acf096e39473294d4840635ca6206fec7"
+
+  bottle :unneeded
+
+  def install
+    system "./install.sh", prefix
+  end
+
+  def caveats; <<~EOS
```

Comment:
We don't usually like long `caveats` blobs like this. If `bats` and `bats-core` can't play nice when installed at the same time, a `conflicts_with` statement would be a little cleaner and catch potential mistakes earlier.

Reply:
As an example:

```ruby
conflicts_with "bats", :because => "both install a `bats` binary"
```